### DISCUSSION TITLE
Multiple nested shapes at the port

### DISCUSSION THREAD

**Discussion by rmnsfx** [clientIO/joint#2616] [Q&A] [answered]
### Introduction

Сan a port have multiple nested shapes? 

Is it possible to specify children in portMarkup?

Is there any example?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment, marked as answer]
Yes. Here's an example: https://www.jointjs.com/demos/element-port-and-link-label-markup

**Reply by rmnsfx** [reply]
Thank you
